.github/workflows/publish.yml

```diff
@@ -10,10 +10,14 @@ env:
 
 jobs:
   publish:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }} 
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-latest]
```

Comment:
Let's specify `macos-10.15` rather than `macos-latest` here - [reference](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on)